tests/cypress/e2e/basicE2E.cy.ts

```diff
@@ -47,15 +43,21 @@ describe('rmrkCollectionItem', () => {
 describe('rmrkGalleryItem', () => {
   it('galleryListedItemActions', () => {
     cy.loginWithKeyring()
-    cy.rmrkGalleryListedItemActions(
-      '9834502-32b209ecbe2b33dd24-DINOXEGGS-DXDANGLS-0000000000000080',
-      'DinoPQ...LPj2rd'
+    cy.visit('/rmrk')
```

Comment:
Is it necessary to visit '/rmrk' first?
In the usual path, we would directly open the specific detail page. WDYT